src/footprintmonitoring/azext_footprintmonitoring/generated/_help.py

```diff
@@ -0,0 +1,238 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['footprintmonitoring profile'] = """
+    type: group
+    short-summary: footprintmonitoring profile
+"""
+
+helps['footprintmonitoring profile list'] = """
+    type: command
+    short-summary: Retrieves the information about all Footprint profiles under a subscription.
+    examples:
+      - name: List Footprint profiles in a Resource Group
+        text: |-
+               az footprintmonitoring profile list --resource-group "rgName"
+"""
+
+helps['footprintmonitoring profile show'] = """
+    type: command
+    short-summary: Retrieves the information about a single Footprint profile.
+    examples:
+      - name: profilesGet
```

Comment:
Example name comes from swagger spec. Is it possible to refine the name better in the swagger spec level and regenerate this extension?

Reply:
fixed. see equivalent file under azext_footprint/*